docs/source/amp.rst

```diff
@@ -0,0 +1,365 @@
+.. role:: hidden
+    :class: hidden-section
+
+Automatic Mixed Precision package - torch.amp
+==================================================
+
+.. automodule:: torch.amp
+.. currentmodule:: torch.amp
+
+``torch.amp`` provides convenience methods for mixed precision:  using fast FP16 arithmetic where you can and
```

Comment:
Need to clarify that "mixed precision" is choosing to run operations in fp16 or fp32, and why you'd choose a different datatype for different operations. Be succinct and try to be mathematically clear. No marketing speak, please.